src/nepiada/env/nepiada.py

```diff
@@ -191,16 +199,62 @@ def step(self, actions):
 
         # The observation returned in 'step' function is the current position of all agents
```

Comment:
This comment needs to be updated as the contents of 'observations' have been changed

Reply:
Don't forget to delete the previous comments that are no longer true. Line 200, 201 in this case